framework/src/main/java/com/github/dbmdz/flusswerk/framework/flow/builder/SetConfigurationStep.java

```diff
@@ -14,7 +14,7 @@
  */
 public class SetConfigurationStep<M extends Message, R, W> {
 
```

Comment:
Wären für Klassenbezeichner nicht Substantive besser als Verben? Zum Beispiel:
```suggestion
public class ConfigurationStepSetter<M extends Message, R, W> {
```
... wobei `Setter` jetzt auch wieder etwas missverständlich ist :thinking: ...

Reply:
Namen sind schwer :thinking: Ich fand `<Aktion>Step` sprechender und klarer.

`ConfigurationStepSetter` müsste wenn dann `ConfigurationSetterStep` heißen, weil es ja der "Step" ist, der die "Configuration" "setzt".

Reply:
...dann könnte man das natürlich auch einfach nur `ConfigurationSetter` nennen und `Step` weglassen.

Reply:
Letzteres (`ConfigurationSetterStep`) fände ich eigentlich am besten - es ist ein Substantiv und klingt nicht nach "Setter / Getter".

Reply:
Wie persönlich besprochen das `Set` aus den Namen entfernt.